### DISCUSSION TITLE
Brew install fails on macos-14

### DISCUSSION THREAD

**Discussion by aokj4ck** [community/community#121133] [Actions]
### Select Topic Area

Bug

### Body

`brew install xcodegen` fails during a GitHub Action workflow running macos-14 (aka macos-latest), with `xcode-select -s /Applications/Xcode_15.3` and having worked previously.

| Command | Error Output |
| -- | -- |
| `brew install xcodegen` | Error: Cannot install under Rosetta 2 in ARM default prefix (/opt/homebrew)!<br>To rerun under ARM use:<br>    arch -arm64 brew install ...<br>To install under x86_64, install Homebrew into /usr/local. |
| `arch -arm64 brew install xcodegen` | arch: posix_spawnp: brew: Bad CPU type in executable |
| `arch -x86_64 brew install xcodegen` | Error: Cannot install under Rosetta 2 in ARM default prefix (/opt/homebrew)!<br>To rerun under ARM use:<br>    arch -arm64 brew install ...<br>To install under x86_64, install Homebrew into /usr/local. |
| `/usr/sbin/softwareupdate --install-rosetta --agree-to-license` | Installing Rosetta 2 on this system is not supported. |

Sample project: https://github.com/mapbox/mapbox-search-ios/pull/217

**Comment by grandmas-favorite** [comment]
Hi @aokj4ck!

Just to clarify - this worked earlier, and you changed nothing, and now it's failing?

**Comment by aokj4ck** [comment]
@grandmas-favorite this issue has been resolved (sorry I forget exactly how and when).
